themes/default/layouts/page/cloud-engineering-days.html

```diff
@@ -1,6 +1,8 @@
 {{ define "hero" }}
     <header class="page-hero">
-        <img class="mx-auto mb-6 w-full" src="/images/cloud-engineering-days/cloud-eng-days-hero.png" />
+        <a href="/cloud-engineering-days/#registration">
```

Comment:
Nit, this works too:

```suggestion
        <a href="#registration">
```

Reply:
ah yeah going too fast for my own good, thanks!